controller/users_controller.py

```diff
@@ -0,0 +1,32 @@
+from model.repository.users_repository import UsersRepository
```

Comment:
This file is no longer needed since we'll go directly from AppController to the Repositories